### DISCUSSION TITLE
How to implement explicit phase field solving?

### DISCUSSION THREAD

**Discussion by Yasunaoka** [idaholab/moose#26929] [Q&A Modules: Phase field] [answered]
Hi all, I have recently been working on implementing a simulation of an impact fracture problem through the coupling of tensor mechanics and phase field. After weeks of struggling with the phase field not converging (reference #26728 ), I decided to try to transform the implicit solution of the phase field into an explicit solution by referring to the tensor mechanics module.

```

[GlobalParams]
  implicit = false
[]

```
```
[TimeIntegrator]
  type = CentralDifference
  solve_type = lumped
[]

```

however the calculations did not turn out as expected: although the mechanics module did calculate and pass the driving force to the phase field module, however, the phase field module did not return any results.

![微信图片_20240229164224](https://github.com/idaholab/moose/assets/147731232/2bb98954-577f-4ff4-9c8e-ff635c9b4973)

If an implicit solution is used (i.e., the two code blocks above are deleted), the correct result can be obtained.So something must have gone wrong in between.Here is a link to my input file
[https://github.com/Yasunaoka/moose/tree/master/modules/combined/problem](url)

Does anyone know how to implement explicit phase field solving? Any suggestions are greatly appreciated!

**Comment by GiudGiud** [comment, marked as answer]
Hello

Did you try a simpler explicit time integrator? Like a 1st order explicit euler?
Central difference has been coded for specific types of solves iirc

@laagesen

**Reply by Yasunaoka** [reply]
Hi, thanks for the advice, although both the ExplicitEuler and ExplicitMidpoint integrators failed, but ActuallyExplicitEuler is working fine for me.
